waku/waku_core/topics/sharding.nim

```diff
@@ -91,15 +72,66 @@ proc weightedShardList*(topic: NsContentTopic, shardCount: int, weightList: seq[
 
   list.sort(hashOrder)
 
-  ok(list)
+  ok(list) ]#
 
-proc singleHighestWeigthShard*(topic: NsContentTopic): Result[NsPubsubTopic, string] =
+#[ proc singleHighestWeigthShard*(topic: NsContentTopic): Result[NsPubsubTopic, string] =
   let count = ? shardCount(topic)
 
-  let weights = biasedWeights(count, topic.bias)
+  let weights = repeat(1.0, count)
 
   let list = ? weightedShardList(topic, count, weights)
 
   let (pubsub, _) = list[list.len - 1]
 
-  ok(pubsub)
+  ok(pubsub) ]#
+
+proc genZeroSharding*(topic: NsContentTopic, count: int): NsPubsubTopic =
```

Comment:
Does this function need to be exported `*` or is it simply internal call of `autosharding*()`?

Reply:
Is there a way to not export when used in testing? That would be ideal.

Reply:
Unfortunately not. If you need to test this, it would indeed have to remain exported.